### PR TITLE
Update ru_RU.lang

### PR DESCRIPTION
**Pull request by Adaptivity**
Also added pack.mcmeta. If you won't change to mcmod.info let me know and I'll revert it.

### CODE REVIEW

File: src/main/resources/assets/chisel/lang/en_US.lang [hunk 1]
```diff
@@ -1218,4 +1218,4 @@ container.chisel.mode=Mode:
 container.chisel.mode.single=Single
 container.chisel.mode.panel=Panel
 container.chisel.mode.column=Column
-container.chisel.mode.row=Row
+container.chisel.mode.row=Row
```
Comment (TheCricket) [inline src/main/resources/assets/chisel/lang/en_US.lang]: What was the difference here?

Comment (Adaptivity) [inline src/main/resources/assets/chisel/lang/en_US.lang]: Deleted empty space. Nothing else.

**Comment by TheCricket** [comment]
Ok, no pack.mcmeta and were all set. Didn't realize the mcmod.info was never changed name wise.

**Comment by Adaptivity** [comment]
Hmm, reason why you won't have pack.mcmeta? I know it's doesn't do anything useful(it actually removes "missing a pack.mcmeta file" from logs) but some mods have it. So why not in Chisel?

**Comment by TheCricket** [comment]
I mean it really doesn't do anything, I will get the others opinions on this

**Pull request merged**